pkg/apps/strategy/interfaces.go

```diff
@@ -4,13 +4,13 @@ import (
 	"strconv"
 	"strings"
 
-	kapi "k8s.io/kubernetes/pkg/apis/core"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
@tnozicka this switch is good as deployer run as external process, so it should use the external client..